r/tools/linuxlibs.R

```diff
@@ -54,7 +54,9 @@ download_binary <- function(os = identify_os()) {
       download.file(binary_url, libfile, quiet = quietly),
       silent = quietly
     )
-    if (!file.exists(libfile)) {
+    if (file.exists(libfile)) {
```

Comment:
This is an unrelated followup fix to the Linux packaging patch that merged yesterday, didn't seem worth its own ticket.